tests/lms/views/content_item_selection_test.py

```diff
@@ -0,0 +1,95 @@
+import pytest
+
+from lms.views.content_item_selection import content_item_selection
+
+
+class TestContentItemSelection:
+    def test_it_passes_the_content_item_return_url_to_the_template(
+        self, pyramid_request
+    ):
+        template_variables = content_item_selection(pyramid_request)
```

Comment:
Again, this name `template_variables` feels a bit verbose to me for something so common. Is there an existing convention we could use? In `TestCanvasFileBasicLTILaunch` and a few others the name `data` is used. In a few places in h the name `result` is. used.

Reply:
I think I prefer something more verbose like `template_variables` or `template_data`. The verboseness is much less of a crime in my mind than variable names like `data`, `result`, etc that tell you nothing about the variable, could equally apply to any variable in the code. You might as well just call it `variable` :)

Perhaps it could be shortened to `template_vars` (although that's the same length as `template_data`) or `tmpl_vars` or something. I've never been much of a fan of those sorts of shortenings though as they just introduce confusion about what "tmpl" is short for.

There might be a better solution but it's not coming to me right now